validator/build.gradle

```diff
@@ -41,6 +41,12 @@ dependencies {
     // yaml reader
     compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.11.1'
 
+    // json reader
+    compile group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1'
```

Comment:
what's the reason to use json-simple instead of jackson? 
jackson is also easy to create map from json. https://mkyong.com/java/how-to-convert-java-map-to-from-json-jackson/ :)

and it's built-in in java, no need to add a dependency.

Reply:
Basically challenge was to convert nested structure like segment documents to a flattened map (since for validation we want to access each field) but not sure we can do that using Jackson. I have tried couple of examples but unfortunately no luck.